tests/component/test_interpreter.py

```diff
@@ -1,13 +1,17 @@
 import pytest
 
 from nidaqmx._base_interpreter import BaseInterpreter
-from nidaqmx._grpc_interpreter import GrpcStubInterpreter
 from nidaqmx.error_codes import DAQmxErrors
 
-try:
-    from nidaqmx._grpc_interpreter import _ERROR_MESSAGES
-except ImportError:
-    _ERROR_MESSAGES = {}
+
+def get_error_messages():
```

Comment:
I don't think this function is necessary. I think it's sufficient to move the import of GrpcStubInterpreter into the try block or into the test function.

Reply:
Removed the get_error_messages() method.